teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorClientOptions.java

```diff
@@ -29,8 +32,22 @@ public class ValidatorClientOptions {
       arity = "1")
   private String beaconNodeApiEndpoint = ValidatorConfig.DEFAULT_BEACON_NODE_API_ENDPOINT;
 
+  @Option(
+      names = {"--Xbeacon-node-ssz-blocks-enabled"},
+      paramLabel = "<BOOLEAN>",
+      description = "Use SSZ encoding for API block requests",
+      hidden = true,
+      showDefaultValue = CommandLine.Help.Visibility.ALWAYS,
+      arity = "0..1",
+      fallbackValue = "true")
+  private boolean validatorClientSszBlocksEnabled = DEFAULT_VALIDATOR_CLIENT_SSZ_BLOCKS_ENABLED;
+
   public void configure(TekuConfiguration.Builder builder) {
-    builder.validator(config -> config.beaconNodeApiEndpoint(parseApiEndpoint()));
+    builder.validator(
+        config -> {
+          config.beaconNodeApiEndpoint(parseApiEndpoint());
+          config.validatorClientUseSszBlocksEnabled(validatorClientSszBlocksEnabled);
```

Comment:
nit: I think we can just chain these:
```suggestion
          config.beaconNodeApiEndpoint(parseApiEndpoint())
                  .validatorClientUseSszBlocksEnabled(validatorClientSszBlocksEnabled);
```

I guarantee to have broken spotless with that...